packages/view/src/Charcoal/View/Twig/TwigEngine.php

```diff
@@ -162,8 +164,16 @@ public function twig(): TwigEnvironment
      */
     public function render(string $templateIdent, $context): string
     {
-        $arrayContext = json_decode(json_encode($context), true);
-        return $this->twig()->render($templateIdent, $arrayContext);
+        try {
+            $arrayContext = json_decode(json_encode($context, JSON_THROW_ON_ERROR), true);
+            return $this->twig()->render($templateIdent, $arrayContext);
+        } catch (JsonException $e) {
+            throw new Exception(
+                sprintf("Twig cannot render \"%s\", Error : \"%s\".", $templateIdent, $e->getMessage()),
```

Comment:
Single quotes are cleaner. No need to quote the error message.

```suggestion
                sprintf('Twig cannot render template [%s]: %s', $templateIdent, $e->getMessage()),
```